Makefile

```diff
@@ -208,11 +200,50 @@ ifeq ($(quickstart), true)
 	./scripts/package_quickstart.sh deploy/$(target)/manifests/$(ver) deploy/$(target)/quickstart/olm.yaml deploy/$(target)/quickstart/crds.yaml deploy/$(target)/quickstart/install.sh
 endif
 
-##########################
-#  OLM - Commands        #
-##########################
+################################
+#  OLM - Install/Uninstall/Run #
+################################
 
 .PHONY: run-console-local
 run-console-local:
 	@echo Running script to run the OLM console locally:
 	. ./scripts/run_console_local.sh
+
+.PHONY: uninstall
+uninstall:
+	@echo Uninstalling OLM:
+	- kubectl delete crd deploy/upstream/quickstart/crds.yaml
```

Comment:
shouldn't this just be `kubectl delete -f deploy/upstream/quickstart/crds.yaml`?

the rest of the crd deletes shouldn't be needed

Reply:
This should also include `kubectl delete apiservices.apiregistration.k8s.io v1.packages.operators.coreos.com` to unblock namespace deletion.

Reply:
Done 👍